packages/cli/src/commands/project/index.ts

```diff
@@ -57,6 +57,8 @@ export default async function main(client: Client) {
     argv = getArgs(client.argv.slice(2), {
       '--next': Number,
       '-N': '--next',
+      '--inspect': Boolean,
+      '-i': '--inspect',
```

Comment:
**suggestion (soft-blocking):** remove the leftover args

```suggestion
```

In this or a follow-up PR, can we remove these leftovers?